@stellar/lyra-api/src/__tests__/index.test.js

```diff
@@ -0,0 +1,9 @@
+import LyraAPI from "../index";
+
+describe("lyra API", () => {
+  it("has keys", () => {
```

Comment:
Added this as a proof of concept that the configuration is working. Strikes me as a good idea to have Jest set up so it's easy to write tests against the public API for the module